File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 C/PRAK504-2410817120021-Aulia-Az-Zahra.c
#include <stdio.h>
int reverse(int n)
{
    //Lengkapi Function ini
    int reversed = 0;
    while (n != 0)
    {
        int remainder = n % 10;
        reversed = reversed * 10 + remainder;
        n /= 10;
    }
    return reversed;
}
int main()
{
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
    return 0;
}

File: MODUL 5 Praktikum Pemrograman/PRAKTIKUM 5 PY/PRAK504-2410817120021-Aulia-Az-Zahra.py
def reverse(n):
    #Lengkapi Function ini
    reversed = 0
    while n != 0:
        remainder = n % 10
        reversed = reversed * 10 + remainder
        n //= 10
    return reversed

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))